gammapy/image/models/core.py

```diff
@@ -170,6 +172,136 @@ def evaluate(lon, lat, lon_0, lat_0, r_0):
         return u.Quantity(norm.value * (sep <= r_0), "sr-1", copy=False)
 
 
+class SkyEllipse(SkySpatialModel):
+    r"""Constant elliptical model
+
+    .. math::
+       \phi(\text{lon}, \text{lat}) =
+                \begin{cases}
+                    N & \text{for }  \,\,\,dist(F_1,P)+dist(F_2,P)\leq 2 a \\
+                    0 & \text{otherwise }\,,
+                \end{cases}
+
+
+
+    where :math:`F_1` and :math:`F_2` represent the foci of the ellipse (located by the method `find_foci()`),
+    :math:`P` is a generic point of coordinates :math:`(\text{lon}, \text{lat})`,
+    :math:`a` is the major semiaxis of the ellipse and N is the model's
+    normalization, in units of :math:`\text{sr}^{-1}`.
+    
+    The model is defined on the celestial sphere, by computing angles and distances with the functions defined in
+    `astropy.coordinates.angle_utilities`, and setting the normalization such that:
+
+    .. math::
+       \int_{4\pi}\phi(\text{lon}, \text{lat}) \,d\Omega = 1\,.
+
+
+    Parameters
+    ----------
+    lon_0 : `~astropy.coordinates.Longitude`
+        :math:`\text{lon}_0`: `lon` coordinate for the center of the ellipse.
+    lat_0 : `~astropy.coordinates.Latitude`
+        :math:`\text{lat}_0`: `lat` coordinate for the center of the ellipse.
+    semi_major : `~astropy.coordinates.Angle`
+        :math:`a`: length of the major semiaxis, in angular units.
+    e : `float`
+        Eccentricity of the ellipse (:math:`0< e< 1`).
+    theta : `~astropy.coordinates.Angle`
+        :math:`\theta`: 
+        Rotation angle of the major semiaxis.  The
+        rotation angle increases clockwise (i.e., East of North) from the positive `lon`
+        axis.
+
+
+        Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        from gammapy.image.models.core import *
+        import astropy.units as u
+        from gammapy.maps import Map, WcsGeom
+        import matplotlib.pyplot as plt
+        from matplotlib import rc
+        rc('font',**{'family':'sans-serif','sans-serif':['Helvetica']})
```

Comment:
This modifies the global rc state of matplotlib, that means if a user does a second plot in the same Python session, those changes will also apply to the second one, which might not be desired. Instead you could use the [rc_context](https://matplotlib.org/api/_as_gen/matplotlib.pyplot.rc_context.html)  context manager. You'll find examples on the linked page, as well as in the Gammapy code e.g. https://github.com/gammapy/gammapy/blob/master/gammapy/maps/base.py#L998